src/Ilios/AuthenticationBundle/Resources/config/services.yml

```diff
@@ -24,3 +24,260 @@ services:
         class: Ilios\AuthenticationBundle\Shibboleth\Authentication
         scope: request
         arguments: ["@request", "@ilioscore.authentication.manager"]
+    security.access.aamc_method_voter:
```

Comment:
Consider pulling these out into a separate voters.yml file to make services easier to work with.